pulpcore/cli/core/context.py

```diff
@@ -38,20 +38,28 @@ class PulpArtifactContext(PulpEntityContext):
     HREF = "artifact_href"
     LIST_ID = "artifacts_list"
     READ_ID = "artifacts_read"
+    CREATE_ID = "artifacts_create"
 
-    def upload(self, file: IO[bytes], chunk_size: int = 1000000, check_exists: bool = True) -> Any:
```

Comment:
We're not using `check_exists=False` anywhere and maintaining it was making the code more and more complex.